test/parallel/test-synchronousworker.js

```diff
@@ -0,0 +1,264 @@
+'use strict';
+
+const common = require('../common');
+
+const {
+  strictEqual,
+  throws,
+} = require('node:assert');
+
+const {
+  SynchronousWorker,
+} = require('worker_threads');
```

Comment:
nit: consistency in the usage of the prefix
```suggestion
} = require('node:assert');

const {
  SynchronousWorker,
} = require('node:worker_threads');
```